app/lib/common/widgets/user_avatar.dart

```diff
@@ -23,7 +23,7 @@ class UserAvatarWidget extends ConsumerWidget {
             ActerAvatar(
               mode: DisplayMode.User,
               uniqueId: client.userId().toString(),
-              size: size,
+              size: data.profile.hasAvatar() ? size : size * 2,
```

Comment:
it really feels like we have a bunch of place where we do this... like ... we should probably take a look at our package and figure out why that is a problem, rather than putting a conditional `size * 2` everywhere it is being used...